pymc3/tests/test_distributions.py

```diff
@@ -1371,24 +1389,36 @@ def test_binomial(self):
     @pytest.mark.xfail(
         condition=(SCIPY_VERSION < parse("1.4.0")), reason="betabinom is new in Scipy 1.4.0"
     )
-    def test_beta_binomial(self):
-        self.checkd(
+    def test_beta_binomial_logcdf(self):
+        self.check_logcdf(
             BetaBinomial,
             Nat,
             {"alpha": Rplus, "beta": Rplus, "n": NatSmall},
+            lambda value, alpha, beta, n: sp.betabinom.logcdf(value, a=alpha, b=beta, n=n),
         )
-        self.check_logp(
+
+    @pytest.mark.xfail(
```

Comment:
This one does not require scipy > 1.4.0 right?

Since it doesn't fail you can also combine it with the test_beta_binomial_discrete_logcdf below.

Reply:
You are right, thanks for spotting